statsmodels/stats/contrast.py

```diff
@@ -55,7 +55,10 @@ def __init__(self, t=None, F=None, sd=None, effect=None, df_denom=None,
                 self.pvalue = self.dist.sf(self.statistic, df_denom)
             else:
                 "normal"
-                self.pvalue = self.dist.sf(np.abs(value)) * 2
+                pvalue = np.full_like(value, np.nan)
+                not_nan = ~np.isnan(value)
+                pvalue[not_nan] = self.dist.sf(np.abs(value[not_nan])) * 2
+                self.pvalue = pvalue
```

Comment:
No objection to be explicit and careful, but are there cases where the 1-line version messes up?  My n=1 test looked the same.

Reply:
What is the 1 line version?

Reply:
GH can't find the diff anymore, but I think I was referring to the status-quo version.